dsms/core/configuration.py

```diff
@@ -74,10 +74,31 @@ class Configuration(BaseSettings):
         description="JWT bearer token for connecting to the DSMS instance",
     )
 
+    reauthenticate: bool = Field(
+        True,
+        description="""Whether to reauthenticate with username and password
+        when the token is expired.""",
+    )
+
     ping_dsms: bool = Field(
         True, description="Check whether the host is a DSMS instance or not."
     )
 
+    ktypes_on_startup: bool = Field(
```

Comment:
Name suggestions:
- fetch_ktypes_on_startup
- auto_fetch_ktypes

Reply:
I would prefer `auto_fetch_ktypes`. The other suggestions is too long in my opinion.